src/main/groovy/org/ods/doc/gen/core/FileSystemHelper.groovy

```diff
@@ -0,0 +1,15 @@
+package org.ods.doc.gen.core
+
+import org.springframework.stereotype.Service
+
+import java.nio.file.Files
+import java.nio.file.Path
+
+@Service
+class FileSystemHelper {
+
+    Path createTempDirectory(String id){
+        return Files.createTempDirectory(id)
+    }
+
```

Comment:
Good point to add also the method to remove the temp folder. Where are you delete the temp folder?

Reply:
Now it is not needed

Reply:
Why?
This is something other team members are asking for.
You need to add a comment somewhere to explain them why it is not needed.
Important details need to be written down.

Reply:
We are not using the code that you request, if needed we can add the code